src/portal/lib/src/image-name-input/image-name-input.component.scss

```diff
@@ -0,0 +1,50 @@
+.clr-form {
+    width:100%;
+}
+
+.selectBox {
+    position: absolute;
+    width: 100%;
+    height: auto;
+    border: 1px solid #ccc;
+    background-color: white;
+    border: 1px solid rgba(0, 0, 0, .15);
```

Comment:
why here need to set 2 border property? I think it should only keep one.

Reply:
I have removed one.